src/scales/scale.linear.js

```diff
@@ -169,16 +170,24 @@
 				this.max++;
 			}
 
-			var niceRange = helpers.niceNum(this.max - this.min, false);
-			var spacing = helpers.niceNum(niceRange / (maxTicks - 1), true);
-			var niceMin = Math.floor(this.min / spacing) * spacing;
-			var niceMax = Math.ceil(this.max / spacing) * spacing;
+			if (this.options.ticks.fixedStepSize && this.options.ticks.fixedStepSize > 0) {
+				var j = this.min;
+				while (j <= this.max) {
```

Comment:
It is probably safer to exit the loop when you've added enough spaces rather than when j > max. The reason is that for very small numbers this may fail.

What about setting `numSpaces = this.options.ticks.fixedStepSize` and then setting `spacing = (this.max - this.min) / numSpaces`. That way you can reuse the loop that's in the else block for both cases.